17.0.Dockerfile

```diff
@@ -230,4 +230,3 @@ ONBUILD ARG DB_VERSION=latest
 ONBUILD RUN /opt/odoo/common/build && sync
 ONBUILD VOLUME ["/var/lib/odoo"]
 ONBUILD USER odoo
-CMD ["sh", "-c", "ls || sleep infinity"]
```

Comment:
Please remove this command on the commit where it was added, not polluting the real diff.